src/librssguard/gui/messagesview.cpp

```diff
@@ -516,6 +516,13 @@ void MessagesView::loadItem(RootItem* item) {
   sort(col, ord, false, true, false, true);
   m_sourceModel->loadMessages(item);
 
+  if (item->toFeed() != nullptr) {
+    if (item->toFeed()->isRTL())
+      setLayoutDirection(Qt::RightToLeft);
```

Comment:
Use full "namespaced" enums like in C++11 class enums. For example "Qt::LayoutDirection::RightToLeft".